### ISSUE TITLE
How to make the code handle oriented bounding box annotations?

### ISSUE THREAD

**Issue by skywalker-ai** [AlibabaResearch/efficientteacher#44]
I have datasets which has annotations which are oriented at an angle, I understand the codebase is designed to handle HBBs, Where are the areas one should change the code to make it work for OBBs ?

**Comment by BowieHsu** [comment]
@skywalker-ai Hello, your question is very valuable. I will try to describe how to perform OBB detection in simple language. First, please take a look at our Dataloader implementation. You will find that we allow annotations other than bbox. Therefore, there is a very tricky method to set kp as1, so that when reading data, you can read the rotation angle in your annotation. After completing this step, you also need to port a detection head and loss function. The format can refer to our standard implementation.

**Comment by skywalker-ai** [comment]
Thank you for your response, I will start working on it using your guidelines.
